### ISSUE TITLE
Some files starting with dot are not added in a new addon created with generator-volto

### ISSUE THREAD

**Issue by wesleybl** [plone/volto#5719]
**Describe the bug**
Although the template has several files that start with a dot, such as [.dockerignore](https://github.com/plone/volto/blob/main/packages/generator-volto/generators/addon/templates/.dockerignore), [.editorconfig](https://github.com/plone/volto/blob/main/packages/generator-volto/generators/addon/templates/.editorconfig), etc., only [.gitignore](https://github.com/plone/volto/blob/main/packages/generator-volto/generators/addon/templates/.gitignorefile) is added.

**To Reproduce**

```
npm install -g @plone/generator-volto
yo @plone/volto test-volto
cd test-volto
yo @plone/volto:addon myaddon
```

**Expected behavior**
All files that start with dot in the template must be in the generated addon

**Software (please complete the following information):**

- OS: Ubuntu 22
- Volto Version : 17.11.5

**Additional context**
It looks like [.gitignore] was added after @instification 's PR: https://github.com/plone/volto/pull/5258

**Comment by instification** [comment]
Hi @wesleybl 

The generator doesn't automatically include dotfiles when copying the template over.

The code that copies over the dotfiles is here: 

https://github.com/plone/volto/blob/4a0580a518756f0cfcbb79fed7d175e84f12453f/packages/generator-volto/generators/addon/index.js#L207-L229

So only the `.github/workflows` `.yarn/releases` and `.gitignore` get copied over at the moment.

I'm afraid I don't know why it doesn't just copy the whole template including ignore files, I suspect it might not be possible to do it that way and all dotfiles will need to be manually added in `index.js`

**Comment by instification** [comment]
@wesleybl yeah my change seems to have stopped the other dotfiles coming through. 

I have made a fix in #5724 which should resolve the issue
